controllers/operatorscaler_controller.go

```diff
@@ -177,6 +186,48 @@ func (r *OperatorScalerReconciler) isOdfDependenciesCsvReady() error {
 	return nil
 }
 
+func (r *OperatorScalerReconciler) createMetrics() error {
+	r.log.Info("entering createMetrics")
```

Comment:
i don't think it's good to couple this log w/ the function name and also no need to mark the location, I see it's being done in other functions but not a good idea.

i don't see the need for logging here at all, if required i'd put a log for `reporting metrics for xyz` in the else.

Reply:
It was requested here https://github.com/red-hat-storage/odf-operator/pull/529#discussion_r1986921384